testsuite/stm32f3/memory.x

```diff
@@ -0,0 +1,6 @@
+MEMORY
```

Comment:
I got this from [here](https://docs.rust-embedded.org/book/start/hardware.html#configuring)

Though above there they say "(There's another RAM region but for simplicity we'll ignore it)", so this may be incomplete?